packages/fiori-freestyle-writer/templates/basic/add/webapp/Component.js

```diff
@@ -1,8 +1,15 @@
+/**
+ * eslint-disable @sap/ui5-jsdocs/no-jsdoc
+ */
```

Comment:
@lboehm Is the `eslint-disable` needed or just the jsdoc below?

Reply:
@devinea Sorry for the late response, I had a busy week. I see you fixed it. Thank you very much!